website/docs/r/machine_learning_compute_cluster.html.markdown

```diff
@@ -110,23 +110,45 @@ The following arguments are supported:
 
 * `vm_size` - (Required) The size of the VM. Changing this forces a new Machine Learning Compute Cluster to be created.
 
-* `identity` - (Required) A `identity` block as defined below. Changing this forces a new Machine Learning Compute Cluster to be created.
-
 * `scale_settings` - (Required) A `scale_settings` block as defined below. Changing this forces a new Machine Learning Compute Cluster to be created.
   
 ---
+* `credential` - (Optional) Credentials for an administrator user account that will be created on each compute node. A `credential` block as defined below. Changing this forces a new Machine Learning Compute Cluster to be created.
 
 * `description` - (Optional) The description of the Machine Learning compute. Changing this forces a new Machine Learning Compute Cluster to be created.
-  
+
+* `identity` - (Optional) A `identity` block as defined below. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `isolated_network_enabled` - (Optional) A boolean value indicating whether network is isolated. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `node_public_ip_enabled` - (Optional)  A boolean value indicating whether enable node public IP address provisioning. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `os_type` - (Optional)  Compute OS Type. Possible values include: `Linux`, `Windows`. Changing this forces a new Machine Learning Compute Cluster to be created.
+
+* `remote_login_port_public_access_enabled` - (Optional)  A boolean value indicating whether enable the public SSH port. Changing this forces a new Machine Learning Compute Cluster to be created.
```

Comment:
i think this would be better as
```suggestion
* `public_ssh_access_enabled` - (Optional)  A boolean value indicating whether enable the public SSH port. Changing this forces a new Machine Learning Compute Cluster to be created.
```
as remote login port means nothing